### ISSUE TITLE
Release a new version

### ISSUE THREAD

**Issue by davidanthoff** [nteract/archived-desktop-app#4836]
Would be great if we could get a release :) Last one is from September, and there is a lot of good stuff that I'd like to see shipping. Updated vega and vega-lite versions first and foremost.

In general, is there maybe a chance to move to a more regular release cycle? Just so that stuff that makes it onto `master` doesn't linger there for months before it actually gets shipped to users? I _really_ love the VS Code release process of just shipping a release every month, no matter what.

**Comment by captainsafia** [comment]
Great question, @davidanthoff!

We actually did introduce a new monthly release cadence as documented [here](https://github.com/nteract/nteract/blob/master/RELEASING.md).

There was an attempt to release last month (December 30th), but there were issues with the electron-build that needed to be resolved so it didn't end up happening.

As a result, the next release will happen on January 27th.

We should probably do a better job of communicating upcoming and current releases on the Twitter. I started this by [tweeting about this week's dev release](https://twitter.com/nteractio/status/1217140610731167744?s=20) and will do similar for upcoming releases.

Thanks for your patience while we work out issues! This next release is sure to delight!
